models/models_1_0_0/models_test.go

```diff
@@ -298,5 +298,48 @@ func TestConvertBitriseConfig(t *testing.T) {
 }
 
 func TestParseFromInterfaceMap(t *testing.T) {
```

Comment:
Rename this method, it's not the one it tests BUT also implement the `TestParseFromInterfaceMap` function's test!!